<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultizoneIndividualTabForm</class>
 <widget class="QWidget" name="MultizoneIndividualTabForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Physical Conditions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Configure the physical Conditions for this zone</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Current Configuration File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentConfigFileLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>&lt;&lt;File&gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Actions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_9">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Configure Physical Conditions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editConfigFileButton">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../toolbarIcons.qrc">
           <normaloff>:/solverIcons/images/script_edit.png</normaloff>:/solverIcons/images/script_edit.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Boundary Conditions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Configure the Boundary Conditions for this zone</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Current Mesh File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentMeshFileLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>&lt;&lt;File&gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Actions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Import New Mesh File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="importMeshFileButton">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../toolbarIcons.qrc">
             <normaloff>:/new/toolbarIcons/images/document_import.png</normaloff>:/new/toolbarIcons/images/document_import.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Configure Boundary Conditions</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editBoundConfigButton">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../toolbarIcons.qrc">
             <normaloff>:/solverIcons/images/script_edit.png</normaloff>:/solverIcons/images/script_edit.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../toolbarIcons.qrc"/>
 </resources>
 <connections/>
</ui>
